recipe/meta.yaml

```diff
@@ -18,7 +18,7 @@
 #       A Public Release for ISIS3.6.1:                        {% set version = "3.6.1" %}
 #       A Release Candidate for ISIS3.6.1:                     {% set version = "3.6.1_RC" %}
 #       A custom build of ISIS3.6.1 for the CaSSIS mission:    {% set version = "3.6.1_cassis" %}
-{% set version = "8.3" %}
```

Comment:
I think there was a reason for just doing the major and minor here. I can't remember why

Reply:
I remember, we can't create tags for releases any more. So when the version is grabbed by ```git_tag: {{ version }}``` it needs to get the 9.0 branch because a 9.0.0_RC2 tag won't exist

Reply:
So this needs to be set to ```9.0``` and we need to add ```.0_RC2``` to the other changed line in this file